vslib/src/SwitchStateBase.cpp

```diff
@@ -1811,6 +1815,17 @@ sai_status_t SwitchStateBase::refresh_read_only(
         }
     }
 
+    if (meta->objecttype == SAI_OBJECT_TYPE_SYSTEM_PORT)
+    {
+        switch (meta->attrid)
```

Comment:
this entire file is in previous PR

Reply:
This is intentional. There is change in create_system_ports(). System port to local port mapping processing which depends on CorePortIndexMap is introduced (line number 2557 - 2594). Since this function is not available till PR #702 is merged, the commit brought this in this PR. Once PR #702 is merged, I'll handle the conflicts and make sure that correct content is in place.